CMakeLists.txt

```diff
@@ -563,26 +567,25 @@ if(__INTEGRATE_OPENCL)
 endif()
 
 if(USE_CUDA)
-
   set_target_properties(
     lightgbm_objs
     PROPERTIES
-      CUDA_ARCHITECTURES ${CUDA_ARCHS}
+      CUDA_ARCHITECTURES "${CUDA_ARCHS}"
```

Comment:
Prevent the following error: 
```
-- Using _mm_malloc
CMake Error at CMakeLists.txt:572 (set_target_properties):
  set_target_properties called with incorrect number of arguments.


CMake Error at CMakeLists.txt:579 (set_target_properties):
  set_target_properties called with incorrect number of arguments.
```

`CUDA_ARCHS` were passed in the following form: `60616270758086878990100120+PTX`.